ssh/ssh.go

```diff
@@ -30,10 +30,14 @@ import (
 type SSHForwardingClient struct {
 	agentForwarding bool
 	*gossh.Client
+	agentForwardingEnabled bool
```

Comment:
Ok, one minor: could you please rename this to: authAgentReqSent or agentForwardingSent , but not Enabled, and also put some code comment: "Used to check if we have already sent the request to the ssh channel".